optaplanner-wb-webapp/src/main/java/org/optaplanner/workbench/client/resources/images/AppImageResources.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.optaplanner.workbench.client.resources.images;
+
+import com.google.gwt.resources.client.ClientBundle;
+import com.google.gwt.resources.client.ImageResource;
+
+public interface AppImageResources extends ClientBundle {
+
+    @Source("optaPlannerIconBlue.png")
+    ImageResource optaPlannerIconBlue();
```

Comment:
@csadilek What's the naming convention guideline here: name them as how they look or as what purpose they do? So optaPlannerBlueIcon.png versus optaPlannerHoverOverIcon.png (or optaPlannerEnabledIcon.png)?